### PR TITLE
Deprecate ossec-init.conf

### PR DESCRIPTION
**Pull request by SpotHound**
Hello team,

This issue introduced required changes in our tests to remove `ossec-init.conf` deprecated file.

Closes https://github.com/wazuh/wazuh-qa/issues/1007 
Closes https://github.com/wazuh/wazuh-jenkins/issues/2243

### CODE REVIEW

File: deps/wazuh_testing/wazuh_testing/tools/__init__.py [hunk 1]
```diff
@@ -17,21 +18,19 @@
...
     WAZUH_SECURITY_CONF = os.path.join(WAZUH_PATH, 'api', 'configuration', 'security', 'security.yaml')
     WAZUH_SOURCES = os.path.join('/', 'wazuh')
     LOG_FILE_PATH = os.path.join(WAZUH_PATH, 'logs', 'ossec.log')
     API_LOG_FILE_PATH = os.path.join(WAZUH_PATH, 'logs', 'api.log')
     if sys.platform == 'darwin':
-        PREFIX = os.path.join('/', 'private', 'var', 'root') 
+        PREFIX = os.path.join('/', 'private', 'var', 'root')
         GEN_OSSEC = None
     else:
         PREFIX = os.sep
```
Comment (BraulioV) [inline deps/wazuh_testing/wazuh_testing/tools/__init__.py:36]: This `if/else` block can be unified with the one in lines 21-24

**Review by BraulioV — CHANGES_REQUESTED** [review]

**Review by BraulioV — DISMISSED** [review]

File: tests/integration/test_wpk/test_wpk_agent.py [hunk 2]
```diff
@@ -49,9 +49,8 @@
 # 2. v4.1.0
 def get_current_version():
     if platform.system() == 'Linux':
-        config_file_path = os.path.join(WAZUH_PATH, 'etc', 'ossec-init.conf')
-        _config = ConfigObj(config_file_path)
-        return _config['VERSION']
+        _version = subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"], stderr=subprocess.PIPE).decode('utf-8')stderr=subprocess.PIPE).decode('utf-8')
+        return _version.replace("v", "")
```
Comment (BraulioV) [inline tests/integration/test_wpk/test_wpk_agent.py:53]:
```suggestion
        _version = subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"], stderr=subprocess.PIPE).decode('utf-8')
        return _version.replace("v", "")
```

File: tests/integration/test_wpk/test_wpk_manager.py [hunk 3]
```diff
@@ -43,9 +43,8 @@
 
 def get_current_version():
     if platform.system() == 'Linux':
-        config_file_path = os.path.join(WAZUH_PATH, 'etc', 'ossec-init.conf')
-        _config = ConfigObj(config_file_path)
-        return _config['VERSION']
+        _version = subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"], stderr=subprocess.PIPE).decode('utf-8')stderr=subprocess.PIPE).decode('utf-8')
+        return _version
```
Comment (BraulioV) [inline tests/integration/test_wpk/test_wpk_manager.py:47]:
```suggestion
        _version = subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"], stderr=subprocess.PIPE).decode('utf-8')
        return _version
```

**Review by BraulioV — CHANGES_REQUESTED** [review]
Check the requested changes. Also, the function `get_current_version` is basically the same both for manager and agents but, the agent's version will fail for Solaris and macOS systems. 

What I mean by this is I would love to see both functions merged in a single one and imported by the test.

File: deps/wazuh_testing/wazuh_testing/wazuh.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        version = check_output([
+          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"
+        ], stderr=PIPE).decode('utf-8')
+
+    return version
+
+
+def get_service():
+    system_platform = platform.system()
+    service = None
```
Comment (BraulioV) [inline deps/wazuh_testing/wazuh_testing/wazuh.py:34]: This is unnecessary. `if/else` blocks don't create new namespaces.

File: deps/wazuh_testing/wazuh_testing/wazuh.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+    if system_platform in ['Windows', 'win32']:
+        service = 'agent'
```
Comment (BraulioV) [inline deps/wazuh_testing/wazuh_testing/wazuh.py:37]:
```suggestion
        return 'agent'
```
`agent` or `wazuh-agent`?

File: deps/wazuh_testing/wazuh_testing/wazuh.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+import platform
+import os
+from wazuh_testing.tools import WAZUH_PATH, LOG_FILE_PATH
+from subprocess import check_output, PIPE
+
+
+def get_version():
+    system_platform = platform.system()
+    version = None
```
Comment (BraulioV) [inline deps/wazuh_testing/wazuh_testing/wazuh.py:17]:
```suggestion
```

**Review by BraulioV — CHANGES_REQUESTED** [review]

File: deps/wazuh_testing/wazuh_testing/wazuh.py (hunk 5 above)
Comment (SpotHound) [reply deps/wazuh_testing/wazuh_testing/wazuh.py:37]: `wazuh-agent`

File: deps/wazuh_testing/wazuh_testing/wazuh.py (hunk 4 above)
Comment (SpotHound) [reply deps/wazuh_testing/wazuh_testing/wazuh.py:34]: hmm yes, you're right, thanks!

**Review by BraulioV — APPROVED** [review]

File: deps/wazuh_testing/wazuh_testing/tools/__init__.py [hunk 7]
```diff
@@ -45,18 +46,31 @@
...
+def get_service():
+    if platform.system() in ['Windows', 'win32']:
+        return 'wazuh-agent'
+
+    else:  # Linux, sunos5, darwin, aix...
+        service = subprocess.check_output([
+          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-t"
+        ], stderr=subprocess.PIPE).decode('utf-8')
+
+    return 'wazuh-manager' if service == 'server' else 'wazuh-agent'
```
Comment (palaciosjeremias) [inline deps/wazuh_testing/wazuh_testing/tools/__init__.py:72]: Screenshot from 2021-02-05 01-02-59
During a test the following problem was identified:
**wazuh-control** responds with a trailing **\n**. which makes the comparison fails and returns **wazuh-agent** for manager installations.

**Review by palaciosjeremias — CHANGES_REQUESTED** [review]

File: deps/wazuh_testing/wazuh_testing/tools/__init__.py (hunk 7 above)
Comment (SpotHound) [reply deps/wazuh_testing/wazuh_testing/tools/__init__.py:72]: Thank you! This should fix it: https://github.com/wazuh/wazuh-qa/pull/1013/commits/4f69e1174d279fb5cff42a013853b95216f3d9ca

**Review by palaciosjeremias — DISMISSED** [review]
LGTM

**Review by BraulioV — APPROVED** [review]

**Pull request merged**